tests/cases/draw_array.amber

```diff
@@ -0,0 +1,54 @@
+#!amber
+# Copyright 2019 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHADER vertex vtex_shader GLSL
+#version 430
+layout(location = 0) in vec3 position;
+void main() {
+  gl_Position = vec4(position, 1.0);
+}
+END
+
+SHADER fragment frag_shader GLSL
+#version 430
+layout(location = 0) out vec4 final_color;
+void main() {
+  final_color = vec4(1.0, 0.0, 0.0, 1.0);
+}
+END
+
+BUFFER position_buf DATA_TYPE vec3<float> DATA
+-1  1 0
+-1 -1 0
+ 1  1 0
+ 1 -1 0
+-1 -1 0
+ 1  1 0
+END
+
+BUFFER framebuffer FORMAT B8G8R8A8_UNORM
+
+PIPELINE graphics pipeline
+  ATTACH vtex_shader
+  ATTACH frag_shader
+
+  VERTEX_DATA position_buf LOCATION 0
```

Comment:
Ok. this case is 1 vertex buffer.

Make sure there are also draw_array tests cases that exercise:
  - case with  multiple vertex buffers
  - case with an index buffer

Reply:
Those cases are covered by this existing case: tests/cases/draw_triangle_list.amber. 
I understand the confusion, the one added test is because there was no other tests as simple. I will add a file listing all the tests that pass in Dawn backend: issue #567

Reply:
I agree about the multiple vertex buffers.
But I don't see any cases there in AmberScript with INDEX_DATA.

I do see two vkscript cases with "draw arrays indexed", including draw_triangle_list_with_index_buffer.vkscript

Thanks for the plan to add list of tests passing in the Dawn backend.  That will help.